app/views/categories/index.html.erb

```diff
@@ -1,23 +1,25 @@
 <h1 class="title">Categories</h1>
-<%= render 'category-form' %>
+<%= render 'category_form' %>
 <% if @categories %>
   <div class="line-spacing">
     <% @categories.each_with_index do |category, idx| %>
-      <%= content_tag :h2, class: 'subtitle js-category-container', id: "category_title_#{idx + 1}" do %>
+      <%= content_tag :div, class: 'subtitle js-category-container', id: "category_title_#{idx + 1}" do %>
         <div class="js-category active">
-          <%= category.name %>
-          <i class="fas fa-pen has-text-dark icon-padding js-edit-category-icon"></i>
-          <% if category.subcategories.any? %>
-            <i class="fas fa-times has-text-grey-light icon-padding" title="Delete all subcategories to allow deletion of this category"></i>
-          <% else %>
-            <%= link_to category_path(category), method: :delete, data: { confirm: "Are you sure? #{category.name} will be deleted." }, title: "Delete category" do %>
-              <i class="fas fa-times has-text-dark icon-padding"></i>
+          <h2>
+            <%= category.name %>
```

Comment:
I would use the `h2` just for the actual title (`<%= category.name %>`), not to wrap everything - you already have the wrapper div. The icons should be sibling elements, not children of `h2`.